system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -5782,6 +5849,283 @@ class CLdapClient : public CInterface, implements ILdapClient
         else
             return -2;
     }
+
+    //Data View related interfaces
+
+    void createView(const char * viewName, const char * viewDescription)
+    {
+        if(viewName == nullptr || *viewName == '\0')
+            throw MakeStringException(-1, "Can't add view, viewname is empty");
+
+        if (stricmp(viewName, "Administrators") == 0 ||
```

Comment:
This should be in an isReservedGroup() function.  Should possibly call it from other places.